src/solvers/dgmulti/sbp.jl

```diff
@@ -312,70 +318,79 @@ function DGMultiMesh(dg::DGMultiPeriodicFDSBP{NDIMS};
   # volume geofacs Gij = dx_i/dxhat_j
   coord_diffs = coordinates_max .- coordinates_min
 
-  if NDIMS==1
-    rxJ = coord_diffs[1] / 2
+  J_scalar = prod(coord_diffs) / 2^NDIMS
+  J = e * J_scalar
+
+  if NDIMS == 1
+    rxJ = J_scalar * 2 / coord_diffs[1]
     rstxyzJ = @SMatrix [rxJ * e]
-  elseif NDIMS==2
-    rxJ = coord_diffs[1] / 2
-    syJ = coord_diffs[2] / 2
+  elseif NDIMS == 2
+    rxJ = J_scalar * 2 / coord_diffs[1]
+    syJ = J_scalar * 2 / coord_diffs[2]
     rstxyzJ = @SMatrix [rxJ * e z; z syJ * e]
-  elseif NDIMS==3
-    rxJ = coord_diffs[1] / 2
-    syJ = coord_diffs[2] / 2
-    tzJ = coord_diffs[3] / 2
+  elseif NDIMS == 3
+    rxJ = J_scalar * 2 / coord_diffs[1]
+    syJ = J_scalar * 2 / coord_diffs[2]
+    tzJ = J_scalar * 2 / coord_diffs[3]
     rstxyzJ = @SMatrix [rxJ * e z z; z syJ * e z; z z tzJ * e]
   end
 
-  J = e * prod(coord_diffs) / 2^NDIMS
-
   # surface geofacs
   nxyzJ = ntuple(_ -> [], NDIMS)
   Jf = []
 
-  is_periodic = ntuple(_ -> true, NDIMS)
+  periodic = ntuple(_ -> true, NDIMS)
```

Comment:
```suggestion
  periodicity = ntuple(_ -> true, NDIMS)
```
For internal consistency